docs/application/tizen-studio-0/native-tools/index.md

```diff
@@ -1,33 +1,30 @@
 # Native Tools
 
-In the Tizen Studio, you can perform all the development activities from the application project creation to testing and packaging.
+This page explains about various native application development tools supported in Tizen Studio. With Tizen Studio IDE, the native app development is simplified, as it supports app development using languages such as C and C++.
+In addition, Tizen Studio provides enhanced tools for building the UI, debugging, and for compiling and building the native app code.
 
-The Tizen Studio provides various tools to help you with the different native application development process phases:
+Tizen Studio and its native application tools enable you to:
+- Maximize the performance of a device to achieve higher throughput for time-sensitive apps, and run computationally intensive applications such as simulations and games.
+- Provide access to platform libraries to manage native activities and access physical device components such as sensors, touch input panels, and Wi-Fi.
+- Provide enhanced debugging, compilation, and integrated build tools. 
 
-- [Managing Projects](managing-projects.md)
+The native application development phases and the tools used in each phase are as follows:
 
-  You can create new application projects with the Project Wizard, and manage the existing applications in the **Project Explorer** view in the Tizen Studio. You can also manage connected devices, and register certificates for your applications to allow them to be published in the application stores.
+|Development Phases |Description|
+|----|:---|
```

Comment:
It's not a good idea to make a table.
An image to show a process is better.

Reply:
I have made the necessary changes, please review and give your valuable comments.